src/effects/VST3/memorystream.cpp

```diff
@@ -0,0 +1,319 @@
+//-----------------------------------------------------------------------------
+// Project     : SDK Core
```

Comment:
I will skip review of these files, which I assume are copied from a trusted third party library without modifications.